modules/nodes-pods-priority-about.adoc

```diff
@@ -44,6 +45,8 @@ A number of critical components include the `system-cluster-critical` priority c
 ** metrics-server
 ** descheduler
 
+* *openshift-user-critical* - You can use the priority class with important pods that can't bound their resource consumption and do not have predictable behaviour of resource consumption. Prometheus pods under `openshift-monitoring` namespace and `openshift-user-workload-monitoring` namespace use `priorityClassName` `openshift-user-critical`. The monitoring workloads use `system-critical` as their first `priorityClass`. But it causes problem when monitoring uses excessive memory and the nodes can't evict them. Monitoring priority drops to give the scheduler flexibility, to move the heavy workloads around and keep critical nodes alive.
```

Comment:
suggestion: 

* *openshift-user-critical* - You can use the `priorityClassName` field with important pods that cannot bind their resource consumption and do not have predictable resource consumption behavior. Prometheus pods under the `openshift-monitoring` and `openshift-user-workload-monitoring` namespaces use the `openshift-user-critical` `priorityClassName`. Monitoring workloads use `system-critical` as their first `priorityClass`, but this causes problems when monitoring uses excessive memory and the nodes cannot evict them. As a result, monitoring drops priority to give the scheduler flexibility, moving heavy workloads around to keep critical nodes operating.